### PR TITLE
Streaming SQL with Apache Kafka and Kerberos authentication

### PR DESCRIPTION
**Pull request by kwart**
This PR updates existing Docker compose simple example and adds a new one with SQL streaming querying Kerberos-protected Kafka topic.

### CODE REVIEW

File: hazelcast-integration/docker-compose/01_apache_kafka_with_kerberos/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Moreover, when Kerberos authentication is involved, SASL-related Kafka properties must be appropriately configured.
+
+```sql
+CREATE MAPPING trades (
+    id BIGINT,
+    ticker VARCHAR,
+    price DECIMAL,
+    amount BIGINT)
+TYPE Kafka
+OPTIONS (
```
Comment (TomaszGaweda) [inline hazelcast-integration/docker-compose/01_apache_kafka_with_kerberos/README.md:48]: Maybe add explanation of the options below - what is for what etc.

**Comment by kwart** [comment]
Thanks, Tomasz, for the review and comments. I'll work on improvements.

**Comment by kwart** [comment]
The comments should be covered in the last commit. I also rebased the PR on top of the `master` branch. Thanks in advance for a re-review.

**Comment by TomaszGaweda** [comment]
Great job!

**Pull request merged**